### PR TITLE
Subject setup page up and running. added a button from the study page…

### PR DESCRIPTION
**Pull request by Blistin**
… to get to this new page. Calendar button does not work 100%

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/3See/etect-app/pull/21" target="_blank">ReviewNinja

File: modules/core/client/config/core.client.routes.js [hunk 1]
```diff
@@ -40,6 +40,10 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
             .state('subject-status', {
                 url: '/subject-status',
                 templateUrl: 'modules/core/views/subject.status.client.view.html'
+            })
+            .state('subject-setup', {
```
Comment (Blistin) [inline modules/core/client/config/core.client.routes.js:44]: !fix

Comment (Blistin) [inline modules/core/client/config/core.client.routes.js:44]: !fixed

Comment (Blistin) [inline modules/core/client/config/core.client.routes.js:44]: !completed

**Comment by Blistin** [comment]
These appear to be the same changes that are being made in the sidebar branch just without the sidebar

**Pull request closed without merging**